src/vernemq_dev_api.erl

```diff
@@ -35,6 +36,53 @@ disconnect_by_subscriber_id(SubscriberId, Opts) ->
             vmq_queue:force_disconnect(QueuePid, normal, proplists:get_bool(do_cleanup, Opts))
     end.
 
+%% @doc Reauthorize subscriptions by username and subscriber_id
+%%
+%% Given a username and subscriber_id all subscriptions
+%% of a matching client are reauthorized against the
+%% currently installed `auth_on_subscribe` hooks.
```

Comment:
Clarify that this also holds for the `_m5` hooks?

Reply:
Fixed